src/components/Slider/Slider.js

```diff
@@ -4,36 +4,30 @@ import BtnSlider from './BtnSlider'
 import data from './dataSlider'
 
 export default function Slider() {
+    const [slideIndex, setSlideIndex] = useState(0)
 
-    const [slideIndex, setSlideIndex] = useState(1)
-
-    // console.log(slideIndex)
+    console.log(slideIndex)
     console.log(setSlideIndex)
 
     const nextSlide = () => {
-        if(slideIndex !== data.length){
+        if(slideIndex < data.length - 4){
             setSlideIndex(slideIndex + 1)
         }
-        else if (slideIndex === data.length){
-            setSlideIndex(1)
-        }
     }
 
     const prevSlide = () => {
-        if(slideIndex !== 1){
+        if(slideIndex > 0){
             setSlideIndex(slideIndex - 1)
         }
-        else if (slideIndex === 1){
-            setSlideIndex(data.length)
-        }
     }
 
     return (
         <div className="container-slider">
-            {data.map((obj, index) => {
+            {
+                data.slice(slideIndex, 4 + slideIndex).map((obj, index) => {
                 return (
                     <div className={"slide"}
```

Comment:
you can move html you return using map method in separated component calls SliderCard as an example